test/operators/second_quantization/test_fermionic_op.py

```diff
@@ -234,14 +234,14 @@ def test_pow(self):
             self.assertFermionEqual(fer_op, targ)
 
     def test_adjoint(self):
-        """Test adjoint method and dagger property"""
+        """Test adjoint method"""
         with self.subTest("adjoint"):
             fer_op = ~FermionicOp([("+N", 3), ("N-", 1), ("--", 2 + 4j)])
             targ = FermionicOp([("-N", 3), ("N+", 1), ("++", (-2 + 4j))])
             self.assertFermionEqual(fer_op, targ)
 
-        with self.subTest("dagger"):
-            fer_op = FermionicOp([("+-", 1), ("II", 2j)]).dagger
+        with self.subTest("adojoint2"):
```

Comment:
Its just a test name of course  maybe you meant `adjoint2` without the o after ad?

Reply:
Sorry, my typo. I'll fix it.